web/src/styles/base/text-colors.scss

```diff
@@ -26,6 +26,10 @@
   color: $accent-cool-lightest !important;
 }
 
+.text-accent-cool-darker {
+  color: $accent-cool-darker !important;
```

Comment:
Is there a reason this has to be `!important`? I see that on hover the color is changing, but I think the right thing to do would be to target the problem instead of just emphasizing this.

Reply:
So the USWDS library has its colors done in a way that this is needed which is why all the colors in here have it this way for them to take effect correctly, initially when I used some of the other button varients this was required to create the effect desired (might not be true in current incarnation). 

I completely hate it because using `!important` is an anti-pattern and want to move away from it but for now consistency with the rest of the file and use against the library seems like a better way to do things

Reply:
I would rather see us override the problem than use `!important`. This worked for me.
```
.text-accent-cool-darker.usa-button:hover {
  color: $accent-cool-darker;
}
```
<img width="406" alt="Screenshot 2023-05-01 at 4 24 32 PM" src="https://user-images.githubusercontent.com/22647707/235524710-6b840975-eeb9-4ce8-8d5d-9861e02f1063.png">